From 7eb2c32d96307f72a9fe54dc373a329725be3a9a Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Thu, 22 Dec 2016 21:58:46 +0000
Subject: [PATCH] OS-5871 i40e version warning does not need to go to the
 console Reviewed by: Joshua M. Clulow <jmc@joyent.com> Approved by: Bryan
 Cantrill <bryan@joyent.com>

---
 usr/src/uts/common/io/i40e/i40e_main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/uts/common/io/i40e/i40e_main.c b/usr/src/uts/common/io/i40e/i40e_main.c
index b52934529b..52a64e6c30 100644
--- a/usr/src/uts/common/io/i40e/i40e_main.c
+++ b/usr/src/uts/common/io/i40e/i40e_main.c
@@ -1226,14 +1226,14 @@ i40e_common_code_init(i40e_t *i40e, i40e_hw_t *hw)
 
 	if (hw->aq.api_maj_ver == I40E_FW_API_VERSION_MAJOR &&
 	    hw->aq.api_min_ver > I40E_FW_API_VERSION_MINOR) {
-		i40e_notice(i40e, "!The driver for the device detected a newer "
+		i40e_log(i40e, "The driver for the device detected a newer "
 		    "version of the NVM image (%d.%d) than expected (%d.%d).\n"
 		    "Please install the most recent version of the network "
 		    "driver.\n", hw->aq.api_maj_ver, hw->aq.api_min_ver,
 		    I40E_FW_API_VERSION_MAJOR, I40E_FW_API_VERSION_MINOR);
 	} else if (hw->aq.api_maj_ver < I40E_FW_API_VERSION_MAJOR ||
 	    hw->aq.api_min_ver < (I40E_FW_API_VERSION_MINOR - 1)) {
-		i40e_notice(i40e, "!The driver for the device detected an older"
+		i40e_log(i40e, "The driver for the device detected an older"
 		    " version of the NVM image (%d.%d) than expected (%d.%d)."
 		    "\nPlease update the NVM image.\n",
 		    hw->aq.api_maj_ver, hw->aq.api_min_ver,
-- 
2.21.0

